src/components/navBar/navBar.js

```diff
@@ -438,6 +441,9 @@ function MdNavItem($mdAria, $$rAF, $mdUtil, $window) {
             'ng-blur="ctrl.setFocused(false)" ' +
             'ng-disabled="ctrl.disabled" ' +
             'tabindex="-1" ' +
+            'role="tab" ' +
+            'aria-label="{{ctrl.navItemAriaLabel}}" ' +
```

Comment:
We don't want to set an empty `aria-label` here as that will be flagged as an invalid aria attribute and may cause issues with certain screen readers.

If no `nav-item-aria-label` is defined, no `aria-label` attribute should be applied. We can do this using the following:
```
'ng-attr-aria-label="{{ctrl.navItemAriaLabel ? ctrl.navItemAriaLabel : undefined}}" ' +
```
I have a few examples of this in https://github.com/angular/material/pull/11441/files#diff-dfc7d490c731a45e66ad22d8efb7456dR358.